graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -405,8 +405,9 @@ def _rename_and_suppress_types_and_fields(
 
     Returns:
         Tuple containing the modified version of the schema AST, the renamed type name to original
-        type name map, and the renamed field name to original field name map. The maps contain
-        entries for all non-suppressed types/ fields that were changed.
+        type name map, the renamed field name to original field name map, and the set of interfaces
+        to make unqueryable because one of their descendants in the inheritance hierarchy was
```

Comment:
At several points in this PR, I mention an inheritance hierarchy to describe how types implement interfaces in GraphQL schemas. Although GraphQL doesn't have inheritance from object types, describing things as an inheritance hierarchy seems like the most straightforward way to describe this structure. Does this seem clear enough from context as-is?